rtf_vis_tool/src/Components/frustum_classes/se3.js

```diff
@@ -27,10 +27,8 @@ class SE3 {
             for (var col = 0; col < 3; col++) {
                 this.transform_matrix.set(row, col, rotation.get(row,col));
             }
-        }
 
-        //set last column to translation array
-        for (var row = 0; row < 3; row++) {
+            //set last column to translation array
```

Comment:
are we able to set the whole row at once now? we don't need the for loop?

Reply:
before we were thinking that wasn't possible in `numjs` right?

Reply:
so unfortunately, we can still only edit matrices one element at a time. We cannot set the whole row at once. The change I made was so that I didn't use a second for loop to add the translation matrix elements.

Reply:
i think this is outside the 2nd loop though, right? so `row` is just defaulting to the last value from the previous loop?

Reply:
do we have a unit test for this constructor, to check that the transform matrix is set correctly?

e.g. for R= 3x3 identity matrix, and translation=[1,2,3]?

Reply:
![image](https://user-images.githubusercontent.com/40643713/115796765-66386980-a3a0-11eb-9106-9f95e22a74d0.png)

So for this code in the se3 constructor, you are right, `row` is just defaulting to the last value from the previous loop because it's setting (`row`, 3) to an element in the translation vector. So still, I'm restricted to only setting one element of the matrix at a time.

Reply:
I'll work on a unit test for se3 constructor so we can make sure the transform matrix is set correctly.

Reply:
oh i see, i was reading it wrong. thanks

Reply:
cool, a unit test can go in another PR, i'll approve now